### ISSUE TITLE
Version 1.7.2 breaks on XCode 8 swift 3.2

### ISSUE THREAD

**Issue by lookatitude** [auth0/Auth0.swift#150]
Guys I've just updated my pods and got version 1.7.2

Being that is a minor release should not break any thing right(semver)?
In this case is not Auth0 that is breaking is the dependency on cryptoSwift 0.7.0 that is to work with XCode 9 and Swift 4. The error is the same as stated in the cryptoSwift issue [#485](https://github.com/krzyzanowskim/CryptoSwift/issues/485)

Reverted back to 1.7.1 and all was fine.

**Comment by cocojoe** [comment]
There is no breaking code change, it's not clear what you are asking. You say that it's broken but provide no screenshots and then say it's not Auth0 but another repo `cryptoSwift`?

The current release doesn't give any guarantees about Xcode 9 / Swift 4, 3.2. I made a PR #149   yesterday now that Xcode 9 GM has been released that makes the lib compatible with both Xcode 8 and 9. I suggest you try the latest "master" and feedback.  Thx
